src/components/AudioPanel/CurrentlyPlayingTrack.js

```diff
@@ -1,17 +1,37 @@
-import { Stack, Typography } from "@mui/material";
+import { Skeleton, Stack, Typography } from "@mui/material";
 import React from "react";
 
 function CurrentlyPlayingTrack({ track }) {
   return (
     <Stack flex={1} direction="row" gap={2} alignItems="center">
-      <img
-        height={56}
-        src={track?.albumImage}
-        alt={track?.name ?? "album"}
-        loading="lazy"
-        style={{ borderRadius: "2px" }}
-      />
-      <Typography variant="span">{track?.name}</Typography>
+      {!track && (
+        <>
+          <Skeleton
+            variant="rectangular"
+            animation={false}
+            width={56}
+            height={56}
+          />
+          <Skeleton 
```

Comment:
2 skeletons? Why?

Reply:
![image](https://github.com/BobMakhlin/music-ui/assets/33604634/01491a45-fd68-46fc-828a-0ba620ed02ee)

The second skeleton is a placeholder for track name :))